### DISCUSSION TITLE
Diagrams.Net Hyperlinks Published

### DISCUSSION THREAD

**Discussion by carlheinmostert** [requarks/wiki#3075] [Help / Questions]
Good day

I really like the diagrams.net integration.

I have noticed that the element hyperlinks do not seems to be clickable in the wiki.
Is there any way to enable this?

Thanks

Carl

**Comment by DraakBZH** [comment]
Same probleme here. If i put 3 links in my diagram, only the last created is clickable. Others no.

wiki.js:  v2.5.201 (running on docker)
